src/Components/Forms/src/EditContextDataAnnotationsExtensions.cs

```diff
@@ -53,15 +53,20 @@ private static void ValidateModel(EditContext editContext, ValidationMessageStor
             messages.Clear();
```

Comment:
/cc @pranavkm given this reaction from the runtime.

Reply:
Thanks @NTaylorMullen. These look great.